cerbero/commands/package.py

```diff
@@ -44,7 +44,7 @@ def __init__(self):
             ArgparseArgument('-f', '--force', action='store_true',
                 default=False, help=_('Delete any existing package file')),
             ArgparseArgument('-d', '--no-devel', action='store_false',
-                default=True, help=_('Do not create the development version '
+                default=False, help=_('Do not create the development version '
```

Comment:
The old behaviour was correct, except that it was saving devel in a variable named no_devel.
So when nothing was set, no_devel = False, when -d was passed store_false was storing false in no_devel.
If you change the behaviour to store no_devel in args.no_devel, so by dedault it's no_devel=True, so also need to change action to store_true, so when -d is passed no_devel=True.